### DISCUSSION TITLE
3D Printing Filament Type for Light Source

### DISCUSSION THREAD

**Discussion by polygonfuture** [srozum/film_camera_tester#9] [Q&A] [answered]
Hi Serhiy,

What filament type did you use to print the calibrated light source?  Did you use PETG, ABS, or something else?
I assume that PLA would deform with the heat from the LED correct?

**Comment by srozum** [comment, marked as answer]
LED runs way below it's rated current in all modes, and if fan and radiator works correctly the temperature should never reach above 50˚C.

Everything was printed using PLA filament, and extensively tested, as well as running LED at full power. 

Also note that there are nylon bushings between a radiator and 3D printed lens housing to provide an air gap. LED and it's copper plate not touching nor held by a 3D printed lens housing. It's glued to a radiator.

Of course, if you want to be on a safer side, use ABS or PETG.

**Reply by polygonfuture** [reply]
Nice great to know.   PLA it is!

**Reply by polygonfuture** [reply]
@srozum You mention it is glued to the heatsink.   What type of glue was used?  Was this something other than a standard thermal paste?
